### PR TITLE
DCS-1932: ✨Add Arrival Age And Confirm Arrival Button To `New Arrival Summary Page` Part 2 

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: server/views/pages/bookedtoday/summary.njk [hunk 1]
```diff
@@ -25,16 +25,12 @@
...
                             },
                             {
-                                key: {  text: "Arriving from" },
+                                key: { text: "Arriving from" },
                                 value: { text: arrival.fromLocation }
-                            },
-                            {
-                                key: {  text: "Court outcome" },
-                                value: { text: "" }
                             }
```
Comment (louisenorris) [inline server/views/pages/bookedtoday/summary.njk:34]: Removed Court outcome as out of scope for current ticket.

File: server/routes/bookedtoday/summaryController.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  public view(): RequestHandler {
+    return async (req, res) => {
+      const { id } = req.params
+      const arrival = await this.expectedArrivalsService.getArrival(id)
+      return res.render('pages/bookedtoday/summary.njk', { arrival })
+    }
+  }
+
+  public redirectToConfirm(): RequestHandler {
+    return async (req, res) => {
```
Comment (connormaglynn) [inline server/routes/bookedtoday/summaryController.ts:18]: Talked to @louisenorris about this we've agreed to temporarily duplicate the `choosePrisonerController` logic here whilst this work is still in progress, and the Summary Page is not integrated into the UX.

Not 100% clear whether this logic will exist only here once the Summary Page is integrated or if there are scenarios where the user will go straight to confirming an arrival without a summary.

Either way, we can either refactor or remove the logic in `choosePrisonerController` when we're crossing that bridge 😄 

Happy to hear third opinions on this, though! 🧠

File: server/utils/utils.test.ts [hunk 3]
```diff
@@ -472,3 +473,10 @@ describe('isFutureDate', () => {
     expect(result).toEqual(false)
   })
 })
+
+describe('calculateAge', () => {
+  test('calculate age from date of birth', () => {
+    const twentyYearsAgo = moment().subtract(20, 'years').format('YYYY-MM-DD')
+    expect(calculateAge(twentyYearsAgo)).toBe(20)
+  })
```
Comment (andrewrlee) [inline server/utils/utils.test.ts:481]: Nice to have some boundary tests here - whats slightly less than 20 years ago, slightly more?

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Pull request merged**